tests/storage/storage_migration/conftest.py

```diff
@@ -267,3 +287,78 @@ def deleted_old_dvs_of_stopped_vms(unprivileged_client, namespace):
         # target DV after migration name is: <source-dv-name>-mig-<generated_suffix>
         if "-mig-" not in dv.name:
             assert dv.delete(wait=True)
+
+
+@pytest.fixture(scope="class")
+def blank_disk_dv_for_storage_migration(namespace, source_storage_class):
+    with create_dv(
+        source="blank",
+        dv_name="blank-dv-for-hotplug",
+        namespace=namespace.name,
+        size=DEFAULT_DV_SIZE,
+        storage_class=source_storage_class,
+        consume_wffc=False,
+    ) as dv:
+        yield dv
+
+
+@pytest.fixture(scope="class")
+def fedora_vm_for_hotplug_and_storage_migration(unprivileged_client, namespace, cpu_for_migration):
+    name = "fedora-volume-hotplug-vm"
+    with VirtualMachineForTests(
+        name=name,
+        namespace=namespace.name,
+        body=fedora_vm_body(name=name),
+        cpu_model=cpu_for_migration,
+        client=unprivileged_client,
+    ) as vm:
+        running_vm(vm=vm)
+        yield vm
+
+
+@pytest.fixture(scope="class")
+def vm_for_storage_class_migration_with_hotplugged_volume(
+    namespace, blank_disk_dv_for_storage_migration, fedora_vm_for_hotplug_and_storage_migration
+):
+    with virtctl_volume(
+        action="add",
+        namespace=namespace.name,
+        vm_name=fedora_vm_for_hotplug_and_storage_migration.name,
+        volume_name=blank_disk_dv_for_storage_migration.name,
+        persist=True,
+    ) as res:
+        status, out, err = res
+        assert status, f"Failed to add volume to VM, out: {out}, err: {err}."
+        wait_for_vm_volume_ready(vm=fedora_vm_for_hotplug_and_storage_migration)
+        yield fedora_vm_for_hotplug_and_storage_migration
+
+
+@pytest.fixture(scope="class")
+def vm_with_mounted_hotplugged_disk(vm_for_storage_class_migration_with_hotplugged_volume):
+    # Mount the disk to the VM and write some data there
+    # # sudo mkfs.ext4 /dev/sda
+    # # sudo mkdir /mnt/hotplug
+    # # sudo mount /dev/sda /mnt/hotplug
```

Comment:
can we remove these comments? I do not see a reason to have the implementation here. if needed, please use proper docstring